app/ide-desktop/lib/cloud/src/authentication/src/api.tsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * @file Module containing the API client for the Cloud backend API.
+ * 
+ * Each exported function in this module corresponds to an API endpoint. The functions are
+ * asynchronous and return a `Promise` that resolves to the response from the API.
+ */
+import { API_URL } from "./config";
+
+
+
+// =============
+// === Types ===
+// =============
+
+interface Request {
+    accessToken: string;
+    path: string;
+    method: HttpMethod;
+    body?: Record<string, string>;
+}
+
+enum HttpMethod {
+    get = "GET",
+    post = "POST",
+    put = "PUT",
+    delete = "DELETE",
+}
+
+export type Organization = {
+    id: string;
+    userEmail: string;
+    name: string;
+}
+
+
+
+// ================
+// === FetchApi ===
+// ================
+
+/// Utility function for performing a REST request to an API endpoint.
+///
+/// Parses the response as JSON (if possible) and returns the result.
```

Comment:
what do you mean by "if possible"? Do you mean that if the response was a valid JSON object?